plugins/capture_space/lib/kCaptureSpaceVersionManager.php

```diff
@@ -18,26 +18,23 @@ class kCaptureSpaceVersionManager{
 	
     private static $config = null;
     
-	private static function initConfig(){
+	private static function getConfig($os, $osTypes, $version = null){
 		if(!self::$config){
 			$filename = __DIR__ . '/../config/' . self::INI_FILE_NAME;
 			self::$config = parse_ini_file($filename, true);
 		}
-		return self::$config;
-	}
-	
-	private static function getConfig($os, $osTypes, $version = null){
-		$config = self::initConfig();
+		
+		self::$config = self::initConfig();
 		if($version){
 			$version = str_replace('.', '_', $version);
-			if(!isset($config[$version])){
+			if(!isset(self::$config[$version])){
 				return null;
 			}
-			$sections = array($version => $config[$version]);
+			$sections = array($version => self::$config[$version]);
 		}
 		else {
-			uksort($config, 'version_compare');
-			$sections = array_reverse($config, true);
+			uksort(self::$config, 'version_compare');
+			$sections = array_reverse(self::$config, true);

```

Comment:
This basically means INI must be written in ascending order of versions (lowest version - first in INI).
I'm OK with that, although it is error prone.
wouldn't be easier to maintain a [latest] section?